### PR TITLE
[ci] Added Github CI Build

### PR DESCRIPTION
**Pull request by devkapilbansal**

### CODE REVIEW

**Comment by devkapilbansal** [comment]
So reordering some classes fix the error. 🤔 Thanks for the help @Purhan 😄

**Comment by Purhan** [comment]
@devkapilbansal The build previously failed because of this change: https://github.com/openwisp/openwisp-users/pull/230.
Also read https://github.com/openwisp/openwisp-users/#django-rest-framework-permission-classes :smile:

**Review by nemesisdesign — APPROVED** [review]

**Review by nemesisdesign — CHANGES_REQUESTED** [review]
Thanks @devkapilbansal, the travis config file has to be removed though, can you please do that?

**Review by nemesisdesign — COMMENTED** [review]
A possible follow up to improve testing: https://github.com/openwisp/openwisp-ipam/issues/85

File: requirements.txt [hunk 1]
```diff
@@ -1,5 +1,5 @@
 # TODO: change this when next version of openwisp_users is released
-openwisp-users @ https://github.com/purhan/openwisp-users/tarball/master
+openwisp-users @ https://github.com/openwisp/openwisp-users/tarball/master
```
Comment (atb00ker) [inline requirements.txt:2]: @Purhan was there a reason for using your fork or was this a mistake? :smile:

**Review by atb00ker — APPROVED** [review]
Thanks, Looks good! :smile:

File: requirements.txt (hunk 1 above)
Comment (nemesisdesign) [reply requirements.txt:2]: I believe it was a leftover

**Review by nemesisdesign — APPROVED** [review]
Great work, thanks @devkapilbansal

**Pull request merged**